app/src/main/kotlin/com/zachklipp/intentsendertool/MainActivity.kt

```diff
@@ -29,8 +28,8 @@ public class MainActivity : Activity() {
   private val mResolveView: IntentTargetView by bindView(R.id.resolve_view)
   private val mNoResultsText: TextView by bindView(R.id.no_results_text)
 
-  private val mLaunchAdapter: ArrayAdapter<LaunchAction> by Delegates.lazy() {
-    ArrayAdapter(this, R.layout.support_simple_spinner_dropdown_item, LaunchAction.values())
+  private val mLaunchAdapter: ArrayAdapter<LaunchAction> by lazy {
+    ArrayAdapter(this, R.layout.support_simple_spinner_dropdown_item, LaunchAction.values)
```

Comment:
`.values` should be a method call: http://blog.jetbrains.com/kotlin/2015/12/kotlin-1-0-beta-3-is-out/#more-3263